### DISCUSSION TITLE
Can't get the region name

### DISCUSSION THREAD

**Discussion by grechnevi** [Pryaxis/TShock#1915] [Q&A]
Can't get the region name with the followed command: 
> /region name

I use this command and it says in the chat:

> Hit a block to get the name of the region

Then
1. I hit a block in a region 
2. I get this text in the chat:
> **You lack permission to build in this region.**

Then when I hit a block outside of a region I get this text in the chat:
> There are no regions at this point, or they are not protected.

### So
How am I supposed to get the name of a region if I "lack permission to build" in it?
Is this a bug?

**Comment by moisterrific** [comment]
I just tested the `/region` and `/region name` commands and didn't run into any problems.

Can you try running the command again as superadmin? Or try `/region allow <your player name> <region name>`. You can also use `/region list` to see the list of regions you’ve defined if you forgot the name.

**Reply by grechnevi** [reply]
As superadmin I can do this.

I admit, I didn't make it clear. Yes, I'm not the owner of the region, but I expected that at least users with `tshock.admin.region` permission can get the other's region name.
